reana_db/models.py

```diff
@@ -133,6 +137,25 @@ def get_user_disk_usage(self, block_size=None):
         except Exception:
             return -1
 
+    def get_user_cpu_usage(self):
```

Comment:
`get_user_disk_usage` should follow the same approach as `get_user_cpu_usage`, just reading the value from DB and calculate it only if it hasn't been calculated before. This will be possible once https://github.com/reanahub/reana-workflow-controller/issues/333 is implemented.